src/messagingsession/MessagingSessionConfiguration.ts

```diff
@@ -27,6 +27,11 @@ export default class MessagingSessionConfiguration {
    */
   reconnectLongBackoffMs: number = 5 * 1000;
 
+  /**
+   * The boolean to indicate if the prefetch feature is turned on or not
+   */
+  prefetchOn: boolean = false;
```

Comment:
Can we make this an enum or list of enums.  Perhaps in the future prefetch messages might be sent when added to new channel or whatever.  Right now it is just on connect